paper/basic_training.tex

```diff
@@ -568,73 +568,104 @@ \subsection{Main steps of a molecular dynamics simulation}
-Otherwise, if a brief simulation in the same ensemble is not performed during the equilibration step immediately prior to production, any period of the production simulation should be ignored where drift is observed in the energies, temperatures, pressures, densities, or other defining state-variables of the ensemble.
-This of course precedes estimation of convergence in property calculation.
-Data collection then falls under the category of correctly obtaining unbiased statistics and convergence, which is covered in a separate Best Practices document (\url{https://github.com/dmzuckerman/Sampling-Uncertainty}). 
+
+\subsubsection{Production}
+
+Once equilibration is complete, we may begin collecting data for analysis, and typically this phase is called ``production''.
+The main difference between equilibration and production is simply that in the production simulation, we plan to retain and analyze the collected data.
+Production must always be preceded by equilibration appropriate for the target production ensemble, and production data should never be collected immediately after a change in conditions (such as rescaling a box size, energy minimizing, or suddenly changing the temperature or pressure) except in very specific applications where this is the goal.
+
+For bookkeeping purposes, sometimes practitioners choose to discard some initial production data as additional equilibration; usually this is simply to allow additional equilibration time after a change in protocol (such as a switch from NVT to NPT), and the usual considerations for equilibration apply in such cases (see \citet{ShellNotes}, lecture on Computing Properties). 
+
+Analysis of production is largely outside the scope of this work, but requires considerable care in computing observables and assessing the uncertainty in any computed properties.
+Usually, analysis involves computing expectation values of particular observables, and a key consideration is to obtain \emph{converged} estimates of these properties --- that is, estimates that are based on adequate simulation data so that they no longer depend substantially on the length of the simulation which was run or on its initial conditions.
+
+A separate Best Practices document addresses these critical issues of convergence and error analysis (\url{https://github.com/dmzuckerman/Sampling-Uncertainty}). 
 For more specific details on procedures and parameters used in production simulations, see the appropriate best practices document for the system of interest.
-\todo[inline, color={yellow!20}]{DLM: Possibly clarify distinction between production and equilibration (just whether data is retained, in some cases)}
+
+\todo[inline, color={yellow!20}]{DLM: Probably need to add something here about how often to store data.}
```

Comment:
This would be nice, but to play Devil's advocate, isn't this highly property and system specific?  We shouldn't go into too much detail if we do this - just one or two sentences.

Reply:
Right, I was just thinking of pointing out several considerations:
- It's not useful to store data more frequently than the autocorrelation time since it doesn't actually get you more independent data to analyze
- You should also factor in how much disk space you have/want to use
- You don't want to throw away uncorrelated data if you might need it for something

Sound reasonable?